packages/paste-website/stories/StyledAnchorHyperlink.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Heading} from '@twilio-paste/heading';
+import {LinkIcon} from '@twilio-paste/icons/esm/LinkIcon';
+import {StyledAnchorHyperlink} from '../src/components/Heading';
+
+export const SectionHeaderAnchor = (): React.ReactNode => {
+  return (
+    <Heading as="h2" variant="heading20">
```

Comment:
Should this be testing `AnchoredHeading`?

Reply:
ah yes, this story hadn't been updated w/ the changes. Thanks for catching this!